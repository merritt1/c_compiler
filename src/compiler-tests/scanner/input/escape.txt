c = "hell\111o";

