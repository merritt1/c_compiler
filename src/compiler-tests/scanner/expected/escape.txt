loc = 0001:0001-0001:0001     text = c                        token = [IDENTIFIER          ]     name = c
loc = 0001:0003-0001:0003     text = =                        token = [EQUAL               ]
loc = 0001:0005-0001:0015     text = "hell\111o"              token = [STRING              ]     val = hell\111o
loc = 0001:0016-0001:0016     text = ;                        token = [SEMICOLON           ]
